src/settings/defaults.py

```diff
@@ -313,8 +328,8 @@ def __setattr__(self,k,v):
 
     # Length of timeout applied whenever an external check that runs a students
     # submission is executed,
-    # for example: JUnitChecker, DejaGnuChecker
-    d.TEST_TIMEOUT=60
+    # for example: JUnitChecker, DejaGnuChecker    
```

Comment:
ws change?

Reply:
@physikerwelt : You worte in May 2017 "ws change?" ... the change was just adding a python comment after d.TEST_TIMEOUT=60 to mention, that all shell scripts have to use "ulimit -t 60" so that they can stop on their own.

Reply:
ws= whitespace. Plese delete the trailing whitespace in line 343